src/components/Header/Header.tsx

```diff
@@ -86,29 +57,10 @@ const Header: React.SFC<HeaderProps> = ({
                   <Icon name={'cogwheel'} color={colors.coolGrey} />
                 </span>
                 {settingsStore.showSettings ? (
-                  <SettingsModal handleCloseSettings={handleCloseSettings} />
+                  <SettingsModal handleCloseSettings={handleToggleSettings} />
```

Comment:
Think handleCloseSettings attribute should be renamed as well

Reply:
Discussed